extensions/amp-consent/0.1/test/test-consent-config.js

```diff
@@ -542,6 +546,30 @@ describes.realWin('ConsentConfig', {amp: 1}, (env) => {
 
   describe('expandConsentEndpointUrl', () => {
     it('support expansion in allowed list', async () => {
+      const config = dict({
+        'consentInstanceId': 'test',
+        'checkConsentHref': '/override',
+        'consentRequired': true,
+        'clientConfig': {
+          'test': 'ABC',
+        },
+        'postPromptUI': 'test',
+      });
+      const consentElement = createConsentElement(doc, config);
+      doc.body.appendChild(consentElement);
+      const fakeConsentStateManager = new ConsentStateManager();
+      fakeConsentStateManager.ampdoc_ = doc.body;
+
+      const stubGetServicePromiseForDoc = env.sandbox.stub(
+        GetService,
+        'getServicePromiseForDoc'
+      );
+
+      stubGetServicePromiseForDoc
+        .withArgs(env.sandbox.match.any, 'consentStateManager')
+        .resolves(fakeConsentStateManager);
+
+      stubGetServicePromiseForDoc.callThrough();
```

Comment:
Can you please add a comment here that we're stubbing ServicePromiseForDoc to retrieve the consentManager, but for all other calls to we want to call the underlying implementation. Thanks!

Reply:
Sounds good.